matmeta/payload_metaclass.py

```diff
@@ -336,8 +336,27 @@ def _add_source(self, metadata):
 
     def _add_people(self, metadata):
         people = []
-        
+        # use full names instead of ids to determine whether to add a new entry
+        # TODO: find a more elegant solution
+        def person_already_added(new_citrine_person):
+            for citrine_person in people:
+                if (
+                    citrine_person.name.family == new_citrine_person.name.family
+                    and 
+                    citrine_person.name.given == new_citrine_person.name.given
+                ):
+                    if new_citrine_person.tags:
+                        new_tags = list(set(new_citrine_person.tags))
+                        if citrine_person.tags:
+                            old_tags = list(set(citrine_person.tags))
+                            citrine_person.tags = old_tags.extend(new_tags)

```

Comment:
What if there are duplicates?

Reply:
Good point.  Set addition should probably be used rather than list extension.